### DISCUSSION TITLE
How do you set the Texture State of a RenderTexture?

### DISCUSSION THREAD

**Discussion by huawei-ahcox** [dfranx/SHADERed#177] [Q&A]
Textures loaded from files have controls for some state like min and mag filters, and wrap modes, but render textures used in a subsequent pass as an input texture don't seem to. Is there a way to control that and what are the defaults?

**Comment by dfranx** [comment]
Unfortunately there's currently no way to change these properties for render textures. This problem would be solved by adding sampler objects. I'll try to add this feature in one of the upcoming updates (I'll try to aim for v1.5 release).

As for default values, both min and mag are set to `linear` as per these lines: https://github.com/dfranx/SHADERed/blob/9af52b45f967e464c6a42d4346d0294864ecd2a0/src/SHADERed/Objects/ObjectManager.cpp#L212-L218

**Reply by huawei-ahcox** [reply]
Sounds good and defaults look right, thanks!
